src/routes/safe/components/Apps/utils.ts

```diff
@@ -22,7 +22,7 @@ export const staticAppsList: Array<{ url: string; disabled: boolean }> = [
   { url: `${gnosisAppsUrl}/synthetix`, disabled: false },
 ]
 
-export const getAppInfoFromOrigin = (origin) => {
+export const getAppInfoFromOrigin = (origin: string): Record<string, any> | null => {
```

Comment:
please change any to unknown

Reply:
I did, but it was complaining about the "unknown" type, I'll try again.

Reply:
Any updates of this ? @fernandomg

Reply:
shame on me that assumed those were all old. I'll have a look